src/core/components/operation-summary-path.jsx

```diff
@@ -34,17 +28,22 @@ export default class OperationSummaryPath extends PureComponent{
       isDeepLinkingEnabled,
     } = operationProps.toJS()
 
+    // Add <wbr> word-break elements between each segment, before the slash.
+    const textWithBreaks = path.split(/(?=\/)/g)
+    for (let i = 1; i < textWithBreaks.length; i += 2) {
+      textWithBreaks.splice(i, 0, <wbr />)
+    }
+
     const DeepLink = getComponent( "DeepLink" )
 
     return(
-      <span className={ deprecated ? "opblock-summary-path__deprecated" : "opblock-summary-path" } 
-        onCopyCapture={this.onCopyCapture}
+      <span className={ deprecated ? "opblock-summary-path__deprecated" : "opblock-summary-path" }
         data-path={path}>
         <DeepLink
             enabled={isDeepLinkingEnabled}
             isShown={isShown}
             path={createDeepLinkPath(`${tag}/${operationId}`)}
-            text={path.replace(/\//g, "\u200b/")} />
+            text={textWithBreaks} />
```

Comment:
[DeepLink.text expects a string](https://github.com/swagger-api/swagger-ui/blob/d2e931f3e1c410da92bcad035d200d9d9ea74a80/src/core/components/deep-link.jsx#L17), so prop validation will fail on this change. `textWithBreaks` is a list of strings and react components.

We have couple of options how to resolve it:

1. allow DeepLink to accept text prop in following type: `string` | [`string` | `React Component`]


2. allow DeepLink to accept text prop in following type: `string` | `React Component`

I think I like second option more as it defines simple prop type and we can use following code:

```jsx
const pathWithBreaks = <>{pathParts}</>;

text={pathWithBreaks} />
```


Can you think of anything better?

Reply:
Second option seems reasonable to me (but I am not experienced with React or JSX).

Reply:
Based on https://reactjs.org/docs/typechecking-with-proptypes.html seems like `PropTypes.node` is what we want, which accepts anything renderable, including both arrays & elements, and strings:
```js
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.
optionalNode: PropTypes.node,
```

Alternative is `PropTypes.oneOf([ PropTypes.string, PropTypes.element ])` which would be exactly `2.` above, I think.

I'll do the former for now, and keep the array.

Reply:
PropTypes.node looks like an idiomatic thing for this case. Good catch!